learn/advanced/known_limitations.md

```diff
@@ -82,3 +82,9 @@ user = 1 OR user = 2 […] OR user = 1500 OR user = 1501 […] OR user = 2000 OR
   ["user = 1", "user = 2", […], "user = 1500", "user = 1501", […], "user = 2000", "user = 2001"]
 ]
 ```  
+
+## Size of numeric fields
+
+**Limitation:** Meilisearch can only exactly represent integers between -2⁵³ and 2⁵³.
```

Comment:
I think we could drop the adverb, no? If we want to keep it, then I would suggest using `accurately` instead.

```suggestion
**Limitation:** Meilisearch can only represent integers between -2⁵³ and 2⁵³.
```

Reply:
I thought that numbers outside that range would actually cause Meilisearch to crash. We definitely need an adverb there.

I always feel that "exactly" is a bit wonky, as if it operates more on a level of emphasis than of "quality of being exact/accurate." 

That said, there's precedent for that usage in a fairly reliable source like Wikipedia, so I'm fine with either. "Accurately" might read better, but "exactly" is probably more technically …accurate?

Reply:
I did think of "accurately", but since this is "technical" information, maybe we should go for "technically accurate" terms?